spec/requests/admin/justifications_spec.rb

```diff
@@ -65,9 +65,13 @@
     end
   end
 
-  describe '#destroy' do
+  describe 'DELETE #destroy' do
     it 'deletes a justification' do
-      expect { delete admin_justification_path(justifications.first) }.to change(Justification, :count).by(2)
+      justification = create(:justification)
+
+      expect do
+        delete admin_justification_path(justification)
```

Comment:
@basmoura I've tried `delete :destroy, id: justification` but I'm getting this error `ArgumentError: unknown keyword: id`. Don't you know why?